unitstatushandler/unitstatushandler.go

```diff
@@ -596,6 +596,13 @@ foundloop:
 	instance.statusChanged()
 }
 
+func (instance *Instance) setInstanceStatus(status []cloudprotocol.InstanceStatus) {
```

Comment:
Usually functions without calling make no big sense from a review point of view. Better to introduce it in the commit where is used.

Reply:
Done, move to where this function is called.